contrib/forms/dap/C_FormDAP.class.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
need the standard header (license, author, copyright) in all these files except for table.sql